Update index.md, rearrange alphabetically and add `dvw`, `dvh`, `lvh`, `lvw`, `svh`, and `svw`. (#31641)
mdn/content: files/en-us/web/css/css_values_and_units/index.md

| `ic`  | Average character advance of a full-width glyph in the element's font, as represented by the "水" (CJK water ideograph, U+6C34) glyph. |
| `lh`  | [`line height`](/en-US/docs/Web/CSS/line-height) of the element.                                                                       |

Root font-relative lengths specify a length in relation to the element's {{CSSxRef(":root", "root element")}} ancestor, such as {{HTMLElement("HTML")}} or {{HTMLElement("SVG")}}.
For example, `rem` is relative to the font size on the root element and `rex` is the x-height of the root element's font.

| Unit   | Relative to                                                                                                                             |
| ------ | --------------------------------------------------------------------------------------------------------------------------------------- |
| `rcap` | Cap height (the nominal height of capital letters) of the root element's font.                                                          |
| `rch`  | Average character advance of a narrow glyph in the root element's font, as represented by the "0" (ZERO, U+0030) glyph.                 |

> | `rem`  | [`font size`](/en-US/docs/Web/CSS/font-size) of the root element.                                                                       |

Reviewer: same comment
```suggestion
| `rem`  | Font size of the root element's font.                                                                       |
```

| `rex`  | x-height of the root element's font.                                                                                                    |
| `ric`  | Average character advance of a full-width glyph in the root element's font, as represented by the "水" (CJK water ideograph, (U+6C34)). |

> | `rlh`  | [`line height`](/en-US/docs/Web/CSS/line-height) of the root element.                                                                   |

Reviewer: same comment as above

| Unit   | Relative to                                              |
| ------ | -------------------------------------------------------- |
| `vb`   | 1% of viewport's size in the root element's [block axis](en-US/docs/Web/CSS/CSS_logical_properties_and_values#block_vs._inline).  |

> | `vh`   | 1% of viewport's height.                                 |

Reviewer: These two should be moved down if we're going in alphabetical order

> | `dvh`  | 1% of the [dynamic viewport's](en-US/docs/Web/CSS/length#relative_length_units_based_on_viewport) height. |
> | `dvw`  | 1% of the [dynamic viewport's](en-US/docs/Web/CSS/length#relative_length_units_based_on_viewport) width. |

Reviewer: These can be linked to "/en-US/docs/Web/CSS/length#dynamic". Also, the link need only be on "dynamic"
```suggestion
| `dvh`  | 1% of the [dynamic](/en-US/docs/Web/CSS/length#dynamic) viewport's height. |
| `dvw`  | 1% of the [dynamic](/en-US/docs/Web/CSS/length#dynamic) viewport's width. |
```
- Similarly, the links below can be changed to /en-US/docs/Web/CSS/length#large, /en-US/docs/Web/CSS/length#small, and /en-US/docs/Web/CSS/CSS_logical_properties_and_values#inline_dimension
- Notice that there should be leading `/` that's currently missing in the slugs below

| Unit   | Relative to                                              |
| ------ | -------------------------------------------------------- |
| `dvh`  | 1% of the [dynamic](/en-US/docs/Web/CSS/length#dynamic) viewport's height. |
| `dvw`  | 1% of the [dynamic](/en-US/docs/Web/CSS/length#dynamic) viewport's width. |

> | `lvh`  | 1% of the [large viewport's](en-US/docs/Web/CSS/length#relative_length_units_based_on_viewport) height. |
> | `lvw`, | 1% of the [large viewport's](en-US/docs/Web/CSS/length#relative_length_units_based_on_viewport) width. |

Reviewer: - The links can be updated to the specific section
- There is a straggling `,` after `lvw`

```suggestion
| `lvh`  | 1% of the [large](/en-US/docs/Web/CSS/length#large) viewport's height. |
| `lvw` | 1% of the [large](/en-US/docs/Web/CSS/length#large) viewport's width. |
```

> | `svh`  | 1% of the [small viewport's](en-US/docs/Web/CSS/length#relative_length_units_based_on_viewport) height. |
> | `svw`  | 1% of the [small viewport's](en-US/docs/Web/CSS/length#relative_length_units_based_on_viewport) width. |

Reviewer: ```suggestion
| `svh`  | 1% of the [small](/en-US/docs/Web/CSS/length#small) viewport's height. |
| `svw`  | 1% of the [small](/en-US/docs/Web/CSS/length#small) viewport's width. |
```
